### PR TITLE
#42 Generate todos.json In The Running Folder.

### LINKED ISSUE #42: Generate todos.json In The Running Folder
At the moment the file ``todos.json`` is always generated in the folder where the application resides (the folder where the ``todos-finder-cli.jar`` is residing).

Instead, it should be generated in the folder where the application is run (the folder which is scanned for todos).

Behaviour:

We are in folder ``/home/apps/self-web`` and we are running the command ``java -jar /home/jars/todo-finder-cli.jar``. The file ``todos.json`` is generated in the folder ``/home/jars``. Instead, it should be generated in ``/home/apps/self-web``.

### PR DESCRIPTION
**Pull request by criske**
fixes #42

tested on win:
in `sef-web` folder ran `java -jar %TODO-CLI%\todo-finder-cli-0.0.5-SNAPSHOT.jar` where sys env ` TODO-CLI` points the location of `todo-finder-cli` jar.

let me know if there are problems on linux.

--
no bounty, was a short fix.

### CODE REVIEW

**Comment by charlesmike** [comment]
@criske thank you for your Pull Request. I'll assign someone to review it soon.

**Comment by charlesmike** [comment]
@amihaiemil I couldn't find any assignee for this task. This is either because there are no contributors with role ``REV`` available or because the project does not have enough funds.

Please, make sure there is at least one available contributor with the required role and the project can afford to pay them.

File: src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java [hunk 1]
```diff
@@ -87,12 +87,7 @@ public URI serialize() {
     private File getFile(final boolean deleteFirst) throws URISyntaxException,
             IOException {
         //folder where application is running.
-        final String parent = new File(JsonTodosSerializer.class
-                .getProtectionDomain()
-                .getCodeSource()
-                .getLocation()
-                .toURI())
-                .getParent();
+        final File parent = new File(System.getProperty("user.dir"));
```
Comment (amihaiemil) [inline src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java:90]: @criske I understand ``user.dir`` is predefined by the JDK and always points to the root dir where the app is running?

**Review by amihaiemil — COMMENTED** [review]
@criske just one question

File: src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java (hunk 1 above)
Comment (criske) [reply src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java:90]: @amihaiemil As far I was remembering, is the folder where `java` command is called.
But anyway, just to be sure, I've tested on linux too. In `self-web` I ran `java -jar ~/todo-cli/todo-finder-cli-0.0.5-SNAPSHOT.jar` and `todos.json` was generated in `self-web` folder.

**Comment by amihaiemil** [comment]
@criske awesome, thanks! :D

**Pull request merged**